mentorship ios/Service/Networking/HomeApiService.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  HomeApiService.swift
+//  Created on 28/06/2020
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import Foundation
+import Combine
+
+class HomeApiService : HomeServiceProtocol {
+    
+    var homeResponseData : HomeModel.HomeResponseData?
+    var profileViewModel = ProfileViewModel()
```

Comment:
Why do we have a reference to the ViewModel in the Services?

Reply:
Cause its a network model. I agree network model should not be in view model.